docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

```diff
@@ -13,8 +13,8 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
-<img src="../../images/checkboxRadio.png" alt="photo of checkboxRadio in Tizen">
-
+<img src="../../images/checkboxRadio.png" alt="photo of radio in Tizen">
+<img src="../../images/checkbox-wearable.png" alt="photo of checkbox in Tizen">
```

Comment:
Would you add the sample link button, please?

Reply:
@singa2000 sample link was in 5.0 version of docs, I moved it to 5.5 too